src/Ombi.Core/Engine/V2/MusicSearchEngineV2.cs

```diff
@@ -20,37 +19,40 @@
 using Ombi.Settings.Settings.Models.External;
 using Ombi.Store.Entities;
 using Ombi.Store.Repository;
-using Artist = Hqub.MusicBrainz.API.Entities.Artist;
 using ReleaseGroup = Ombi.Core.Models.Search.V2.Music.ReleaseGroup;
 
 namespace Ombi.Core.Engine.V2
 {
     public class MusicSearchEngineV2 : BaseMediaEngine, IMusicSearchEngineV2
     {
-        private readonly IMusicBrainzApi _musicBrainzApi;
         private readonly ISettingsService<LidarrSettings> _lidarrSettings;
         private readonly ILidarrApi _lidarrApi;
 
         public MusicSearchEngineV2(IPrincipal identity, IRequestServiceMain requestService, IRuleEvaluator rules,
             OmbiUserManager um, ICacheService cache, ISettingsService<OmbiSettings> ombiSettings,
-            IRepository<RequestSubscription> sub, IMusicBrainzApi musicBrainzApi, ISettingsService<LidarrSettings> lidarrSettings,
+            IRepository<RequestSubscription> sub, ISettingsService<LidarrSettings> lidarrSettings,
             ILidarrApi lidarrApi)
             : base(identity, requestService, rules, um, cache, ombiSettings, sub)
         {
-            _musicBrainzApi = musicBrainzApi;
             _lidarrSettings = lidarrSettings;
             _lidarrApi = lidarrApi;
         }
 
         public async Task<ReleaseGroup> GetAlbum(string albumId)
         {
-            var g = await _musicBrainzApi.GetAlbumInformation(albumId);
+            var lidarrSettings = await GetLidarrSettings();
+            Task<AlbumLookup> lidarrAlbumTask = null;
+            if (lidarrSettings.Enabled)
+            {
+                lidarrAlbumTask = _lidarrApi.GetAlbumByForeignId(albumId, lidarrSettings.ApiKey, lidarrSettings.FullUri);
+            }
+            var albumResult = await lidarrAlbumTask;
             var release = new ReleaseGroup
             {
-                ReleaseType = g.ReleaseGroup.PrimaryType,
-                Id = g.Id,
-                Title = g.Title,
-                ReleaseDate = g.ReleaseGroup.FirstReleaseDate,
```

Comment:
The task could be null, so an exception could be thrown here when lidarr is not enabled